src/m4a.c

```diff
@@ -18,7 +18,7 @@ u32 MidiKeyToFreq(struct WaveData *wav, u8 key, u8 fineAdjust)
     u32 val1;
     u32 val2;
     u32 fineAdjustShifted = fineAdjust << 24;
-
+    
```

Comment:
It's recommended to remove any trailing space. So if a line holds nothing, it's supposed to be empty.